examples/blackjack/play.py

```diff
@@ -0,0 +1,21 @@
+import game
```

Comment:
Warto chyba nazwać ten plik `main.py`, bo wtedy jest jednoznaczne, że to jest główny punkt wejścia do programu i będzie spójnie z prostym symulatorem.